test/e2e/tests/phishing-detection.spec.js

```diff
@@ -124,6 +124,10 @@ describe('Phishing Detection', function () {
 
         // Ensure we're not on the wallet home page
         await driver.assertElementNotPresent('[data-testid="wallet-balance"]');
+
+        // Check that the user has been directed to the appropriate warning screen
+        const currentUrl = await driver.getCurrentUrl();
+        assert.equal(currentUrl, 'chrome-extension://invalid/');
```

Comment:
This if course will not work on Firefox

Reply:
Maybe we can check that the current URL is still the warning page OR it's this URL. It'd be the warning page on Firefox, and on older versions of Chrome. Both still demonstrate this was successful.

Reply:
> This if course will not work on Firefox

:man_facepalming: 

I swear I made a note to myself before I stashed my changes on this hours ago, and then unstashed committed and pushed without thinking after coming back to it

Your suggestion is good, thanks